api-reference/10 UI Components/dxChart/1 Configuration/commonAxisSettings/minorGrid/minorGrid.md

```diff
@@ -20,4 +92,9 @@ The **commonAxisSettings**.[minorGrid](/api-reference/10%20UI%20Components/dxCha
 
 Axis-specific settings override common settings.
 
-[note]Displaying the minor grid without the major grid may impair the readability of a chart. To prevent this situation, we recommend you use the minor grid only in conjunction with the major grid.
\ No newline at end of file
+[note]
+
+- Displaying the minor grid alone may impair chart readability. We recommend you implement the minor grid in conjunction with the major grid.
```

Comment:
```suggestion
- If you display the minor grid alone, it may impair chart readability. We recommend you implement the minor grid in conjunction with the major grid.
```